### DISCUSSION TITLE
different permission by location

### DISCUSSION THREAD

**Discussion by donakhseputa** [spatie/laravel-permission#2239] [Q&A]
I have rules of access like this,
1. Each user can access more than 1 branch.
2. Each user can have different permission on a specific branch.

for example:
1. John can access Branch A and Branch F.
2. In Branch A, John has permission to access reporting menu.
3. In Branch F, John has permission to access reporting menu and user management menu.
4. Jane only can access Branch A.
5. In Branch A, Jane only has permission to view the user list.

how to achieve this? and how to write it into the application.

**Comment by erikn69** [comment]
Seems like teams, read Teams Feature on documentation
